packages/boiler-task-selenium/src/spawn-process.js

```diff
@@ -42,12 +42,14 @@ export default function(opts, config, cb) {
     const binPath = join('bin', 'wdio');
     let binaryPath;
 
+    // Try to resolve the path or use the external wdio dependency
     try {
       const webdriverBase = path.dirname(
         require.resolve('webdriverio')
       );
 
-      binaryPath = join(webdriverBase, binPath);
+      // 'webdriverio/build/bin' --> 'webdriver/bin'
+      binaryPath = join(webdriverBase, '..', binPath);
     } catch (err) {
       binaryPath = addroot('node_modules/webdriverio', binPath);
     }
```

Comment:
@stephenkao if you want to dynamically find the `bin` path we can `require` the `package.json`..

```js
const {bin} = require('webdriverio/package');
const [name] = Object.keys(bin);
binaryPath = join(webdriverBase, bin[name]);
```

and don't forget to do something similar in the `catch` just in case `webdriverio` doesn't get put in the root `node_modules` but rather inside the `boiler-task-selenium` `node_modules`